R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -906,6 +910,37 @@ predicted <- predict(model, df)
 head(predicted)
 ```
 
+#### FP-growth
+
+`spark.fpGrowth` executes FP-growth algorithm to mine frequent itemsets on a `SparkDataFrame`. `itemsCol` should be an array of values.
+
+```{r}
+items <- selectExpr(createDataFrame(data.frame(items = c(
+  "T,R,U", "T,S", "V,R", "R,U,T,V", "R,S", "V,S,U", "U,R", "S,T", "V,R", "V,U,S",
+  "T,V,U", "R,V", "T,S", "T,S", "S,T", "S,U", "T,R", "V,R", "S,V", "T,S,U"
+))), "split(items, ',') AS items")
```

Comment:
perhaps it's slightly less clear, since there are 3 references to "items" (or really, just the SparkDataFrame and its column name), which "items" L923 is referring to?